### ISSUE TITLE
skin settings?

### ISSUE THREAD

**Issue by TIAOPINGZHONGGUO** [LTVA1/klystrack#4]
There is an error in changing the skin, and it cannot be changed normally!

**Comment by LTVA1** [comment]
Just tested and it works with all the themes. Please update the whole klystrack folder. Download the latest pre-release and replace the whole folder (make a backup if you have your own songs, instruments etc. there) and try it.

**Comment by TIAOPINGZHONGGUO** [comment]
Problem: But he still can't replace the wrong one!
Description: When your mouse arrow moves to the (Theme) options, they will disappear instantly, and the options bar will all become blank without fonts!
System: The Windows7 64-bit operating system I use!
![IMG_20230311_091726](https://user-images.githubusercontent.com/84202693/224456846-f9d8d0b9-ee0f-410b-8aaa-f2ab80e84918.jpg)
![IMG_20230311_091805](https://user-images.githubusercontent.com/84202693/224456850-0f05c48e-d5bb-48d7-8cfc-716e4ddae923.jpg)

**Comment by LTVA1** [comment]
Hmm, never has the issue. Although I can imagine Windows 7 causing this. Also had some complaints about volume point envelope not working properly — user couldn't drag dots. Maybe this is somewhat similar.

**Comment by LTVA1** [comment]
By the way, is this issue present in original klystrack? iirc when I upgraded to newer SDL library some things stopped working and I was able to fix them eventually. Looks like this is one of the problems. It looks like SDL abandoned Windows 7 or whatever

**Comment by ADG664** [comment]
Okay, after my attempts, the same problem occurred in the old version!

**Comment by LTVA1** [comment]
Hmmm, seems like it has some trouble loading the theme, like it tries but fails (because fonts also disappear, as you say). In several hours I will give you a link of a debug build, it will open second console window when launched. Please try the theme change on the debug version and paste here all the text the console prints. Thus I will at least see if it knows it fails.

**Comment by LTVA1** [comment]
As I promised, [debug build](https://cdn.discordapp.com/attachments/768808912790749216/1162080988785020990/debug.zip)

**Comment by ADG664** [comment]
However, he is still in his original state; I wonder what went wrong?

Uploading 1000059403.mp4…

**Comment by ADG664** [comment]
However, he is still in his original state; I wonder what went wrong?

https://github.com/LTVA1/klystrack/assets/84202693/08fba65e-9a93-4b38-ae65-5e993dfe5f63

**Comment by LTVA1** [comment]
I don't see you actually changing the theme there

**Comment by LTVA1** [comment]
This is very strange. Other menu options work, but this doesn't...

**Comment by ADG664** [comment]
Can I see this operation video?

**Comment by ADG664** [comment]
Video file corrupted?

**Comment by LTVA1** [comment]
No, but it looks really strange. Like other menu entries work but not the theme changer

**Comment by ADG664** [comment]
It seems that you need to download it before you can watch it?

**Comment by LTVA1** [comment]
Yes

**Comment by ADG664** [comment]
This bug is really strange. When you move the mouse pointer to the fault option, the window tool bar font will quickly disappear!
